### ISSUE TITLE
Support multinode deployments

### ISSUE THREAD

**Issue by gbraad** [rdo-infra/weirdo#7]
Currently tested deployments are based on a single node, either using the standard all-in-one or a scenario.

**Comment by dmsimard** [comment]
So, this is a tricky one.
The scope of WeIRDO is to run gate jobs outside of the gate to improve our CI coverage.

Right now, none of the implemented projects (packstack, puppet-openstack, kolla) have multi-node jobs upstream.

I do not want to implement custom jobs in WeIRDO. If we want to test multi node, it will need to follow the right path and be done upstream first.

I know it is possible and @EmilienM looked into it a while back for puppet openstack jobs but it gets a bit hard.

We can definitely keep this issue opened for when upstream starts testing multi node.

**Comment by gbraad** [comment]
I agree with your viewpoint that this is something that needs to be upstream; however for both PackStack and Kolla a multi-node deployment is possible to realize. Just not so sure how this can be achieved and implemented at the OpenStack infrastructure.

**Comment by dmsimard** [comment]
I believe upstream multinode will be facilitated by zuul v3 -- at which point they actually intend to get rid of Jenkins as it will no longer be necessary. There's more details available about it here: https://specs.openstack.org/openstack-infra/infra-specs/specs/zuulv3.html

We're talking Ocata timeframe for Zuul v3 unless significant effort is put towards it in the meantime.
